src/app.js

```diff
@@ -19,6 +21,8 @@ app.use(
   })
 );
 
+app.use(prometheus);
```

Comment:
we should probably add option to pass the route i.e.
```
app.use(prometheus("/metrics"))
```
I know I wrote this, but it feels a bit strange that we have the metrics route bundled in

I think the prom client thing is global so we could probably do this:
```
app.use(prometheus)
app.get("/metrics", (_, res) => {
  res.set("Content-Type", client.register.contentType);
  res.end(client.register.metrics());
});
```

Reply:
as mentioned below, change this to:
```
app.use("/api/v1/{{.Route}}", prometheus)
```